README.md

```diff
@@ -197,11 +205,31 @@ You can combine this with the example data (see above) to run a single build usi
 
 ``` bash
 snakemake --cores 2 \
-    --configfile config/gisaid.yaml \
+    --snakefile segment-focused/Snakefile \
     --config 'inputs=[{"name": "example", "metadata": "example_data/gisaid/metadata.tsv", "sequences": "example_data/gisaid/sequences_{segment}.fasta"}]' \
     -pf test_target
 ```
 
-### clade labeling
+## clade labeling
 
 If you'd like to run clade labeling, you will need to install [LABEL](https://wonder.cdc.gov/amd/flu/label/) yourself. This pipeline assumes that LABEL is located in `avian-flu/flu-amd/`, and should work if you install it into the `avian-flu` directory. If you do not need to label clades, then you can delete `rule add_h5_clade`, the function `metadata_by_wildcards`. You will need to make sure that all references to `metadata` in the pipeline are referencing `metadata_subtype_segment`, not `metadata-with-clade_subtype_segment`, which is generated by `rule add_h5_clade` and adds a new column to the metadata file with clade information.
+
+## Running analyses in a separate working directory
+
+
+We can run the avian-flu worfklows from analysis directory/directories separate to this repo. Imagine the following directory structure:
+
+```
+├── avian-flu
+│   ├── README.md
+│   ├── segment-focused
+│   │   ├── Snakefile
+│   │   └── config.yaml
+│   └── ... (lots more files)
+└── analysis_dir
+```
```

Comment:
_non-blocking_

Maybe worth noting that through the work you've done to resolve paths, it is also possible now to specify arbitrary analysis directories _within_ the repo. 

This makes it possible to run with the `nextstrain build` command 
```
nextstrain build . -s genome-focused/Snakefile --directory test
```
The outputs will be within `avian-flu/test/`

Reply:
I've added a note about the `--directory` argument to the README, but felt it was too much to propose having this inside the `avian-flu` repo to users. It's totally possible, and for those who know how to do it it may be useful. The `.gitignore` will keep them out of version control so this may be quite ergonomic for some people.